### PR TITLE
Feature/search prescriptions simple

### PR DESCRIPTION
**Pull request by Stefann11**

### CODE REVIEW

**Review by velicko-tam — APPROVED** [review]
PR is approved. Good Clean code principles were applied.

File: PatientWebApplication/PatientWebApplication/Controllers/PrescriptionController.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        }
+
+        /// <summary> This method is calling <c>PrescriptionService</c> to get list of all <c>Prescription</c>.  </summary>
+        /// <returns> 200 Ok with list of all prescriptions. </returns>
+        [HttpGet]       // GET /api/prescription
+        public IActionResult Get()
+        {
+
+            List<Prescription> result = PrescriptionService.GetAll();
+            return Ok(result);
```
Comment (trki293) [inline PatientWebApplication/PatientWebApplication/Controllers/PrescriptionController.cs:36]: The list of prescription can be passed as a parameter to the 'ok()' method

File: PatientWebApplication/PatientWebApplication/Controllers/PrescriptionController.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        }
+
+        /// <summary> This method is calling <c>PrescriptionService</c> to get list of all patient <c>Prescription</c>. </summary>
+        /// <returns> 200 Ok with list of all patient prescriptions. </returns>
+        [HttpGet("patient")]       // GET /api/prescription/patient
+        public IActionResult GetPrescriptionsForPatient()
+        { 
+            List<Prescription> result = PrescriptionService.GetPrescriptionsForPatient(1); //idPatient set to 1 no login, change after
+            return Ok(result);
```
Comment (trki293) [inline PatientWebApplication/PatientWebApplication/Controllers/PrescriptionController.cs:45]: The list of prescription can be passed as a parameter to the 'ok()' method

**Review by trki293 — CHANGES_REQUESTED** [review]
It is necessary to change the parts of the code indicated in the comments.

**Review by trki293 — APPROVED** [review]
PR is approved. Clean Code good practices are followed.

**Pull request merged**